go.mod

```diff
@@ -54,14 +54,14 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
 	google.golang.org/grpc v1.30.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.18.6
+	k8s.io/api v0.19.2
+	k8s.io/apimachinery v0.19.2
```

Comment:
Title says 1.20.0-alpha1?

Reply:
@saschagrunert any idea why the version looks off in go.mod?

Reply:
We overwrite the version below, I do not have any influence on this version selection :D